src/transformers/models/layoutlm/modeling_layoutlm.py

```diff
@@ -913,10 +964,140 @@ def forward(
         )
 
 
+@add_start_docstrings(
+    """
+    LayoutLM Model with a sequence classification head on top (a linear layer on top of the pooled output) e.g. for
+    document image classification tasks such as the `RVL-CDIP <https://www.cs.cmu.edu/~aharley/rvl-cdip/>`__ dataset.
+    """,
+    LAYOUTLM_START_DOCSTRING,
+)
+class LayoutLMForSequenceClassification(LayoutLMPreTrainedModel):
+    config_class = LayoutLMConfig
+    pretrained_model_archive_map = LAYOUTLM_PRETRAINED_MODEL_ARCHIVE_LIST
+    base_model_prefix = "layoutlm"
```

Comment:
this is already inherited from `LayoutLMPreTrainedModel` I think so no need to add it here (probably the same applies for other `LayoutLMFor...`)